src/Networking/networking.py

```diff
@@ -112,22 +112,22 @@ def start_connection(self):
 			raise NotImplementedError
 
 	def send_data(self, packet, encoded_status = True):
-
+		# Attempts to encode and send a packet to the target IP:PORT
 		try: 
 			if not encoded_status:
 				self.logger.debug("%s: Packet encoded as type 'ascii'" %(self.netType))
```

Comment:
The recommended method for passing strings to Python's loggers is `debug(<format string>, [args])`. See [here](https://stackoverflow.com/questions/54367975/python-3-7-logging-f-strings-vs) for discussion.

We should fix wherever applicable.

Reply:
Based on that discussion it seems that %s is valid and even represents the optimized approach as documented in https://docs.python.org/3/howto/logging.html#optimization though it is less readable. Since there's not a ton of logging it's probably fine as-is either way.

Reply:
The current implementation does old style string interpolation ([example here](https://realpython.com/lessons/old-school-string-formatting-python/)). The optimized version from the docs is different.
```python
debug('key: %s' % 1)  # eager, old style evaluation; string gets interpolated before function
```
optimized version
```python
debug('key %s', 1)  # lazy evaluation; string is interpolated later
```

Reply:
Totally missed the % interpolation at the end, will fix